Jenkinsfile

```diff
@@ -588,6 +588,17 @@ pipeline {
                         buildHipClangJobAndReboot(setup_flags: MLIR_flags, build_env: extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx908", mlir_build: "ON")
                     }
                 }
+                stage('Fp32 OpenCL MLIR gfx908') {
+                    when {
+                        beforeAgent true
+                        expression { params.TARGET_GFX908 && params.DATATYPE_FP32 }
```

Comment:
This change is valid, but looks inconsistent. Please also merge from the recent develop.

Reply:
> Please merge from the recent develop.

I will do that for you.